packages/backend/src/db/house/house.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types, Schema as MongooseSchema } from 'mongoose';
+import { User } from '../user/user.schema';
+
+export type HouseDocument = House & Document;
+
+@Schema()
+export class House {
+  _id: MongooseSchema.Types.ObjectId;
+
+  @Prop()
+  name: string;
+
+  @Prop()
+  email: string;
+
+  @Prop()
+  address: string;
+
+  @Prop()
+  code: string;
+
+  @Prop()
+  owner: string;
```

Comment:
shouldn't this be an object id? 

```suggestion
@Prop({ type: MongooseSchema.Types.ObjectId, ref: 'User' })
owner: MongooseSchema.Types.ObjectId
```

Reply:
We are referencing firebaseId not the document id because that will be globally available after a user logs in

Reply:
Hmm I see where you're coming from. I'm not sure if mongodb lets you reference keys other than the _id. I think it would be best to change it to reference the _id